queries/robot/highlights.scm

```diff
@@ -1,21 +1,53 @@
-(argument (dictionary_variable) @string.special)
-(argument (list_variable) @string.special)
-(argument (scalar_variable) @string.special)
-(argument (text_chunk) @string)
+(comment) @comment
 
-(keyword_invocation (keyword) @function)
+(section_header) @keyword
+(extra_text) @comment
 
-(test_case_definition (name) @property)
+(setting_statement) @keyword
+
+(variable_definition (variable_name) @variable)
 
-(keyword_definition (body (keyword_setting) @keyword))
 (keyword_definition (name) @function)
+(keyword_definition (body (keyword_setting) @keyword))
 
-(variable_definition (variable_name) @variable)
+(test_case_definition (name) @property)
 
-(setting_statement) @keyword
+(keyword_invocation (keyword) @function.call)
+(ellipses) @punctuation.delimiter
 
-(extra_text) @comment
-(section_header) @keyword
+(argument (text_chunk) @string)
+(argument (scalar_variable) @variable)
+(argument (list_variable) @variable)
+(argument (dictionary_variable) @variable)
```

Comment:
Does it need to be inside an argument to be a variable? Maybe the actual variables can just be variables alone and you can group them together